cohortextractor/query_language.py

```diff
@@ -1,7 +1,28 @@
+from collections import namedtuple
+
+
+_OPERATOR_MAPPING = {
+    "equals": "__eq__",
+    "not_equals": "__ne__",
+    "less_than": "__lt__",
+    "less_than_or_equals": "__le__",
+    "greater_than": "__gt__",
+    "greater_than_or_equals": "__ge__",
+    "on_or_before": "__le__",
+    "on_or_after": "__ge__",
+    "is_in": "in_",
+    "not_in": "not_in",
+}
+
+
 def table(name):
     return Table(name)
 
 
+def category_group(source, comparison, value):
```

Comment:
I guess we can get rid of this once we add operator overloading and then the API becomes even simpler.